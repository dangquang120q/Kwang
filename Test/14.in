21
60
230396376 704492251 692476339 164887418 556017663 870928909 913795859 135681402 699168600 693885881 237210952 251753584 687876154 370563831 124902908 308879781 666244153 370088678 67716930 434298571 690675515 962507772 782324439 787774526 760387577 500510969 290504472 599612787 211850187 83990003 103231262 286784563 493007444 848100217 561959443 814035007 357518280 799705662 903685325 442267053 363375978 25214740 877508131 593963492 841193506 709380155 380488838 668958804 378707073 529542403 581459383 249331730 752823741 928948127 727389216 126182293 509012195 186419455 75962876 181473970 
57
462511307 460424754 313586573 679225628 897469309 807293941 23802808 415741757 831223158 24612514 179467836 144414077 74069070 225143848 475114804 386569861 294652904 924869455 240742865 827106081 68946662 333068566 158547052 995322655 377498740 108046802 440723345 797663556 717706281 162963264 327570267 681604611 877501897 278216591 944854309 935881667 440498466 888684588 553280797 400693702 84377751 176371715 604375785 287362363 982840997 358864002 743551731 786615903 156972831 831251787 261837206 122579951 709034091 162971006 692346131 350414031 80298943 
11
815281906 585451937 595450474 909359777 489524894 694910742 243049721 502721456 675126309 34255120 57138382 
43
328754080 766844725 372693605 588055819 345691866 791620080 140286631 115932418 864344952 985696613 440942354 47340329 941302114 621109564 612542546 951954237 406437645 274759826 524574037 429035890 995319338 628570661 745071882 455036623 106313614 500734145 721250498 770912451 389383634 669366148 896145371 806686887 569084695 459507461 828086354 555697838 913662377 520030416 622648156 457226476 816205690 329033499 568489412 
9
767255226 538971489 175157950 39853973 212993599 995977006 495477947 95596944 680079154 
14
55601025 103409035 948559591 398949681 996791053 702333382 182494973 933797250 897572770 951730925 484895074 243506811 58307601 539868863 
41
528964605 162263345 557677961 626589120 798615358 525060453 723338257 109393924 682714529 653313798 117494251 127579459 267855755 356560797 629213725 140370387 261961377 987745250 31646259 375562487 797198220 654307727 480832051 558431679 546635932 65204834 763991011 996893861 15596444 578209373 198376597 152837959 139489006 489119714 534337385 672410824 780127245 79980541 647892030 973577012 254578699 
14
84937033 911904910 998842356 610052425 854834188 855943089 798225815 903268503 966158673 162988228 613985994 374440502 972255964 149301472 
63
876970139 539715032 396361286 363583808 242380540 709531281 626839807 209048891 65564683 81854691 986147127 39270243 437623401 130307137 398436445 510458939 707700238 656477432 310493029 679707930 108711984 564307253 266047515 909793084 904930330 462331255 824732530 832257119 177342008 297581264 566681224 330507828 814798104 924539798 199774540 701623312 44647899 513151845 390955155 201133859 344330148 426121798 438741523 678852678 555729308 913891670 298780359 491088261 501494528 281089394 411005805 514093679 557769963 401081481 44402754 487363267 835580467 917612252 16634847 149201476 772042116 232333101 264868356 
84
121589645 833736208 219412639 311398538 985787114 255263594 690099660 550362771 634827314 838284753 178281459 595157038 674626004 180115571 746733243 36746356 53219771 139961667 724815552 729641226 50729279 997715845 909346855 864081357 575706465 765974243 785445160 295451213 83680860 376760451 96709968 926364530 274641181 11877376 141164311 226788380 677308374 290377416 645693151 771650189 128610520 252879752 451776769 16296018 462389898 317417850 90068640 676402144 668534016 951413143 965843195 909188401 292059440 498816950 362253445 757446458 223158145 168051525 351150854 246518631 14647290 689473287 720573607 679022100 986680221 137703304 498895266 270642666 797974235 455676111 752291113 463439089 36235474 277437993 933188405 398662210 165538839 420982821 102850272 397314681 260170450 276681729 839189122 382952365 
82
191267284 500798342 128588743 634401461 184460350 366129301 53904714 318457593 474554712 963821791 363877338 163078286 982719787 148995082 973573294 657529547 596362492 802388983 382228605 205180239 536422828 916437494 737736880 992087788 814504782 272629298 110634286 702939201 768249531 99037876 580238952 664111426 987699112 524139896 38238104 846309085 206775275 917889774 981525603 276337208 636303544 56978550 103406591 678324299 212651143 527087079 970106753 764215718 709207788 726790641 233094916 367570060 985357524 680240577 238905645 719227573 770197266 125631223 792802047 44993507 214238257 512254764 86230319 469050213 937580160 934380401 95378871 74502746 422442030 705215025 205510329 79572318 45660999 737513199 478102927 153544252 721158446 488800528 271982137 301489993 507514287 683490733 
85
264005413 872709760 370587907 558053256 440343502 405682331 543706989 104589870 595111902 276872734 36960876 144986027 468573607 561164626 458160596 230191397 345908950 990162131 920120088 155357611 205841801 37636500 586400411 138017152 126689329 420872372 191822027 531954661 268197649 584983859 466660875 111129295 25730684 285349273 38973787 304184112 906046571 852100138 960609794 31822781 664825189 363830887 593364419 378729413 681707653 452137055 343072262 396350166 765422878 54383332 925659861 547252167 119094740 303173803 459255364 469461485 841696212 412392508 642399404 319720335 24181736 387627407 199139749 835867624 437556996 391456781 753390441 743736627 884586162 270779165 879756292 368482352 590891686 46257542 172262063 548097406 625712829 834223570 91579701 59064174 776936714 342768156 220973358 175845185 411030095 
82
834256961 65079822 809145363 242930558 939328021 240623240 370738866 556312073 683075040 294673189 356187210 542474656 514975326 21011239 220534949 493710195 344038947 541552215 668758218 518572993 335789339 503923167 586345012 989307198 808495394 384919096 558724821 287366716 55319048 137188232 212307904 879287052 375032391 90183893 856298117 256327256 741845949 805237012 75466882 277858092 681826781 573574012 91449125 961997565 445433208 758999543 511181271 846681818 19277927 770914614 827257738 584912263 449362151 725438881 840541929 583630010 117514010 573075029 82860238 644909393 829791078 348368621 955682210 838927570 673656722 186054862 163758258 267707848 377074439 302516088 615895740 310062810 747873574 109841923 338814236 658943529 71585192 297490636 593102060 932312647 223125620 799284329 
86
100682261 362221661 977206605 371419482 333362548 845285422 520073114 298233975 899770805 800483350 506248867 836632955 495823209 752475563 310320939 488208632 171768777 861709202 578335487 101393808 407079978 869013874 582053896 465799423 764052721 235027822 93415413 334516134 883640693 316890317 188262061 698870411 78869100 493514315 70624644 768590453 345443946 405029057 82806168 637641097 905524302 283641674 995312095 230663539 110336337 350573194 551472267 454352117 920281399 627610256 387171784 497490804 93008150 26186333 247205484 146830593 415261908 736935668 16489769 21594481 826116272 449639974 227619461 662217557 321305353 215191610 699375358 814186437 85589072 155802048 860133977 454543290 297847766 91199392 251726753 742761624 182549958 574980302 899415370 623189377 499527696 808233256 510052566 316216133 621914287 560158730 
50
205916481 933454659 54706344 259986287 920902269 121953193 930951052 978028296 194983509 628516844 785221866 711595896 706964715 936643720 351745244 691520383 607069359 56503262 132983134 670590544 960180146 162000473 109483655 409424629 238811516 842639946 754125200 552766580 243741506 335002738 15169540 358608782 732268202 162419214 488628362 59205681 686803347 638966759 600500029 771910002 199858244 403302114 254830740 138473721 674652835 926856661 873006568 661124226 468066475 208518790 
62
813450036 78530457 485526480 277194666 301606407 796711321 294487472 477557810 218471975 722732368 887957081 886763764 681148262 99523392 286970978 324026641 158070669 987227410 195884247 297987274 993512169 183953501 304188202 581796815 323168076 595423105 706565522 263364671 306252856 675059544 120870608 446331449 260287469 51077137 221222003 705165866 75619126 22028845 343090654 187603655 339370174 238411654 423349989 456167350 139527393 431618399 679102636 306618949 295403732 893121949 993762329 332130441 58353088 853638197 776370691 269495032 849129049 904412385 267682502 639884449 982734778 577692159 
32
12409441 502268072 86677215 978053351 522443087 304672845 233570648 241320309 949441690 985584388 740001366 114084514 280771628 401852327 14774538 356638261 703820225 573726141 619867448 167766599 53054876 350525527 805984110 532094264 71245509 108918677 845909027 249788574 340533532 361878530 550032657 218859672 
46
257209820 89087693 86969144 778870648 619761630 210302626 893427675 315113278 598082288 659836374 711189011 880740904 18462140 674104456 430289455 765802519 234821111 202400345 677190615 814615822 749075109 43255168 340523165 423605226 868176425 84576455 205675502 853557005 314127573 235320745 73431752 36931705 532629513 718274818 735758725 413719381 671779810 231456420 800150452 185854142 495831477 623387337 603957522 173779808 897982835 296597734 
63
468936409 576447604 440247714 68345843 168490658 834418091 924405656 602511266 487504151 103111132 203833765 817141577 992302008 452503253 519594049 955822084 230265367 223603172 573245538 978876076 158084737 335032837 673534886 375852716 306304335 224342620 84081775 463834438 632130617 768970148 428838728 501264276 700397865 768583626 959529015 100603710 245658662 937574542 972802482 781961930 535288819 245261002 414747307 666098486 559854555 146970146 364260381 88957634 751223661 836706430 542096416 643652336 44803678 203508117 978064819 157821064 130513091 882561180 446426323 339838012 372267467 128421983 85111069 
27
259892325 569360445 46652224 101394565 706703606 179937652 35561551 158368491 373562342 982618414 683958765 720193496 154511588 99365128 709487597 492883062 228889148 590026104 168686653 946477075 316119209 783413097 249164221 317582697 601931704 251978375 206876618 
64
474371926 525343275 375133903 325414170 318294789 63527270 75043622 447283314 20583452 821869112 396907995 191627117 737857290 167915438 194017316 991650014 776511915 464508234 23215799 358364322 435652347 784739317 423878726 85491452 939073620 495136796 306563616 295802976 760587224 614421687 915623780 98471739 769666406 562291102 111090661 919987045 901611206 332316877 728104058 814812084 983927363 189087232 289346939 155526404 219358562 727959668 346554571 185729376 458862272 254420346 609528268 333211076 793681602 57037824 242229606 274602325 487294646 948134494 351253958 909705581 933022303 952603222 674659552 243582804 
