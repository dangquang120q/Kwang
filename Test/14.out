-1489715538
-1371949335
-1875414871
-1361997108
-1851655910
-1397885727
-1336751173
-1357710303
-1466668119
-1324784116
-1329822872
-1418535510
-1387980323
-1402167197
-1420115134
-1320465388
-1381887867
-1622815882
-1345724393
-1582458710
-1366786697
